src/utils/options.js

```diff
@@ -52,17 +49,20 @@ const passwordDescription = `Specifies a source for your secret password. Lisky
 const prettyDescription =
 	'Prints json in pretty format rather than condensed. Has no effect if json option is false. You can change the default behaviour in your config.json file.';
 
+const tableDescription =
+	'Sets output to table format (default). You can change the default behaviour in your config.json file.';
```

Comment:
Prefer `Prints output in table format (default)` as in prettyDescription. Same for `jsonDescription`